src/ui/preferences-window/tabs/GeneralTab.swift

```diff
@@ -150,4 +153,8 @@ class GeneralTab: NSViewController, PreferencePane {
             let _ = LSSharedFileListInsertItemURL(loginItems, kLSSharedFileListItemBeforeFirst.takeRetainedValue(), nil, nil, itemUrl, nil, nil).takeRetainedValue()
         }
     }
-}
\ No newline at end of file
+    
+    private func hideMenubarIconCallback(_ sender: NSControl) {
+        App.statusItem.isVisible = (sender as! NSButton).state == .off
```

Comment:
So many apps require to restart the whole app to remove the menubar icon. I never thought it could be so easy! Amazing and so fast to switch on/off!